superflore/generators/ebuild/ebuild.py

```diff
@@ -157,13 +157,11 @@ def get_ebuild_text(self, distributor, license_text, die_msg=None):
         ret += " license\n\n"
 
         # EAPI=<eapi>
-        ret += "EAPI=" + self.eapi + "\n\n"
-        """
-        @todo: don't hard code this
-        """
+        ret += "EAPI=" + self.eapi + "\n"
+        ret += "PYTHON_COMPAT=( python{2_7,3_5} )\n\n"
```

Comment:
Python3 will not work with all packages, so don't blindly put it everywhere. We need a list of the packages known working with python3.